### PR TITLE
fix: Sequoia form template admin interface now matches Figma designs

### LINKED ISSUE #4783: fix: match Form Templates admin interface to Figma designs
# Bug Report

## User Story
As an admin, I want Form Template related options to match Figma designs so that I can efficiently create and manage forms using the new Form Templates feature efficiently and as originally intended.

## Details
There are a small handful of quick fix issues remaining to ensure that the Form Templates admin interfaces are complete, polished and match Figma designs. These issues are laid out in the acceptance criteria of this issue. 

## Bug Type
- [ ] This bug describes functionality that once worked as expected in version X.X.X.
- [x] This bug describes functionality that never worked as expected.
- [ ] I am not sure whether this functionality ever worked as expected.

## Visuals
See Figma: https://www.figma.com/file/jaVXDA4wjQD9OqTi0l9KQe/Form-Templates?node-id=1882%3A44&viewport=-259%2C1586%2C0.5

## Acceptance Criteria
- [x] Default custom amount text test to "Custom Amount"
- [x] Form templates use accurate thumbnails
- [x] Spacing around "Available Form Templates" header is increased (match Figma)
- [x] Thumbnail aspect ratio increased
- [x] Padding between thumbnails increased
- [x] Improve padding of What is Form Template? copy
- [x] Improve styling of Form Template survey notice
- [x] Prefix introduction section with "Step 1:"
- [x] Conditionally hide introduction fields when introduction section is disabled
- [x] Improve featured image URL input to be full width
- [x] Prefix Payment Amount section with "Step 2:"
- [x] Prefix Payment Information section with "Step 3:"
- [x] Prefix Thank You section with "Step 4:"
- [x] Improve Thank You image description
- [x] Make social sharing fields conditional (based on enable/disable social sharing option)
- [x] Move “Default GiveWP Styles” to “Advanced” settings
- [x] Change the name of the Display Options tab to be “Default Options”
- [x] Change “Display” sub tab to “Form Fields”
- [x] Remove “Floating Labels” as a global option with backwards compatibility

### PR DESCRIPTION
**Pull request by henryholtgeerts**
## Description
Resolves #4783 
This PR introduces minor style and functionality improvements for the Form Template admin interface. Previously, the admin interface was not fully aligned with styles found in the Figma designs. Now, the Form Template admin interface is consistent with the Figma designs, and expected functionality (conditional fields, rearranged admin settings). 

## Affects
This PR affects the Sequoia form template class, the Sequoia form template options configuration file, the Legacy form template class, and GiveWP form assets (images, CSS and JS). 

## What to test
Explore the admin interface, with different settings enabled/disabled. Do screens match Figma designs? When no form is activated, does the templates list interface match Figma designs? Do fields conditionally hide/reveal themselves in an expected way?

## Screenshots:
Screen Shot 2020-06-04 at 9 07 39 PM

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

**Comment by henryholtgeerts** [comment]
The following fatal error has been identified, present in this PR and in the epic/form-templates branch. Currently, this error causes the PR to fail Travis tests, and is a bug separate from the acceptance criteria of this PR.

Screen Shot 2020-06-05 at 8 16 39 PM

File: includes/admin/settings/class-settings-display.php [hunk 1]
```diff
@@ -41,64 +41,30 @@ public function __construct() {
...
-							'id'      => 'css',
-							'type'    => 'radio_inline',
-							'default' => 'enabled',
-							'options' => array(
-								'enabled'  => __( 'Enabled', 'give' ),
-								'disabled' => __( 'Disabled', 'give' ),
-							),
-						),
-						array(
-							'name'    => __( 'Floating Labels', 'give' ),
```
Comment (ravinderk) [inline includes/admin/settings/class-settings-display.php:67]: @henryholtgeerts `Floating Labels` setting removed without backward compatibility. I will suggest moving this setting in `Advanced -> Deprecated Settings`. Add a description to this section which will say that this section contains deprecated settings and will be moved in the future.

@DevinWalker Any thought on above suggestion

Comment (DevinWalker) [reply includes/admin/settings/class-settings-display.php:67]: I'd rather remove it entirely with backwards compatibility. Meaning, the global setting will no longer appear but if you have it enabled and update the existing legacy forms will still display floating labels.

Comment (ravinderk) [reply includes/admin/settings/class-settings-display.php:67]: @DevinWalker got it. if someone wants to disable it then they can disable it from legacy form template setting.

**Pull request merged**